src/main/java/com/carhouse/controller/AjaxRestController.java

```diff
@@ -0,0 +1,29 @@
+package com.carhouse.controller;
+
+import com.carhouse.model.stub.CarModelFilter;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class AjaxRestController {
+
+    @GetMapping("/carMake/{carMakeId}/carModels")
```

Comment:
Very strange endpoint. What is it for?

Reply:
By default, it is not possible to choose a car model if car make is not selected and when car make is selected I make it possible to choose a car model of the selected car make
this method returns list car models of the selected car make